src/applications/vaos/new-appointment/components/DateTimeSelectPage/index.jsx

```diff
@@ -52,10 +48,48 @@ function ErrorMessage({ facilityId, history }) {
           call your local VA medical center
         </NewTabAnchor>
         .
+      </>
+    );
+  }
+
+  return (
+    <>
+      To schedule this appointment, you can{' '}
+      <NewTabAnchor
+        href={`/find-locations/facility/vha_${getRealFacilityId(facilityId)}`}
+      >
+        call your local VA medical center
+      </NewTabAnchor>
+      .
+    </>
+  );
+}
+
+function ErrorMessage({ eligibility, facilityId, history }) {
+  const { request: isRequest } = eligibility;
+  const dispatch = useDispatch();
+
+  return (
+    <div
+      aria-atomic="true"
+      aria-live="assertive"
+      className="vads-u-margin-bottom--2"
+    >
+      <InfoAlert
+        status="error"
+        level="2"
+        headline="We’ve run into a problem trying to find an appointment time"
+      >
+        {renderContent({ dispatch, isRequest, facilityId, history })}
```

Comment:
Typically when we want the specific property from an object we write it like 
`const { request } = eligibility`

What is the advantage of writing out `const { request: isRequest } = eligibility` ?

Reply:
The property is being renamed too 'isRequest'. I prefer 'is' prefix for boolean variables.